src/SpectatorView.Native/SpectatorView.Compositor/SharedHeaders/CompositorShared.h

```diff
@@ -12,8 +12,6 @@
 // FrameProvider type - Exactly 1 of these should be true:
 //TODO: Set this to true if using a USB 3 external BlackMagic Shuttle capture card.
 #define USE_DECKLINK_SHUTTLE    FALSE
```

Comment:
Is this something that's used anywhere? Might also be dead code

Reply:
I'm going to keep this for now. There is a display mode choice affected by this flag that needs further investigation. I've opened https://github.com/microsoft/MixedReality-SpectatorView/issues/59 to track the work.